packages/cli/src/commands/remove-plugin/removePlugin.test.ts

```diff
@@ -32,10 +27,12 @@ import {
   removePluginFromCodeOwners,
 } from './removePlugin';
 
+const mockFs = require('mock-fs');
```

Comment:
move this to imports at the top?

`import mockfs from 'mock-fs'`?

Reply:
By using `import` typescript is complaining that the property `load` does not exist on `mock-fs`.

Reply:
@ayshiff maybe the types are outdated? Seems to be in the https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mock-fs/index.d.ts

Reply:
Try updating the `mock-fs` package and `@types/mock-fs`